build.sh

```diff
@@ -1,117 +1,6 @@
-#!/usr/bin/env bash
+#!/bin/bash
+dotnet tool restore
```

Comment:
Similar to the above, maybe this which disables telemetry, etc? https://github.com/cake-build/resources/blob/master/dotnet-tool/build.sh